extensions/amp-animation/amp-animation.md

```diff
@@ -551,17 +365,17 @@ Both `var()` and `calc()` polyfilled on platforms that do not directly support t
 </amp-animation>
 ```
 
-Animation components can specify their own variables as `--var-name` fields. These variables are propagated into nested animations and override variables of target elements specified via stylesheet (`<style>` tag). `var()` expressions first try to resolve variable values specified in the animations and then by querying target styles.
+Override variables of target elements specified in the `<style amp-custom>` tag by using `--var-name` fields in `amp-animation` component. `var()` expression first try to resolve values specified within the animation component and then query target styles.
 
-### CSS extensions
+#### CSS extensions
 
-`amp-animation` provides several CSS extensions for typical animations needs: `rand()`, `num()`, `width()`, `height()`, `x()` and `y()`. These functions can be used everywhere where CSS values can be used within `amp-animation`, including timing and keyframes values.
+The `amp-animation` component provides the following CSS extensions: `rand()`, `num()`, `width()`, `height()`, `x()` and `y()`. The allowed CSS extensions are valid everywhere where CSS values are usable within the `amp-animation` definition. This includes timing and keyframe values.
```

Comment:
Feel free to push back, but I think "keyframe" as a singular doesn't really make sense in the animation context - they are always plural. I'd still say "keyframes values" even if it reads a little weird.

Reply:
SGTM